crates/electrum/tests/test_electrum.rs

```diff
@@ -272,32 +292,91 @@ fn scan_detects_confirmed_tx() -> anyhow::Result<()> {
     // Mine some blocks.
     env.mine_blocks(101, Some(addr_to_mine))?;
 
-    // Create transaction that is tracked by our receiver.
+    // Broadcast transaction to mempool.
     env.send(&addr_to_track, SEND_AMOUNT)?;
+    env.wait_until_electrum_sees_block()?;
```

Comment:
We are sending tx to mempool, but we are waiting to see a block? Is there a method that waits for electrum to see a certain tx in mempool? If not, we should add it.